src/test/regress/pg_regress_multi.pl

```diff
@@ -509,6 +509,9 @@ sub generate_hba
 
     # we disable some restrictions for local objects like local views to not break postgres vanilla test behaviour.
     push(@pgOptions, "citus.enforce_object_restrictions_for_local_objects=false");
+
+	# we disable propagating create role statements to not have roles as distributed objects during vanilla tests.
+    push(@pgOptions, "citus.enable_create_role_propagation=false");
```

Comment:
This feels unexpected given `ErrorIfGrantRoleWithInheritOrSetOption` is called after  `FilterDistributedRoles` ?

Reply:
I think using enable_create_role_propagation = false might hide some actual bugs, so I'm really not sure if we should pass it.

Instead, I'd rather change `ErrorIfGrantRoleWithInheritOrSetOption` to `WarnfGrantRoleWithInheritOrSetOption` and rely on `EnableUnsupportedFeatureMessages`.

And, make sure to add HINT to WarnfGrantRoleWithInheritOrSetOption such that remind the user should run the same command on the workers

Reply:
Okay, sounds good, let me do that.
I will also extend the PR with some REVOKE tests.

Reply:
It turns out that we cannot run GRANT ROLE statements from the worker nodes. So, I will leave it as erroring out for now unless you have a different idea on this?

Reply:
you cannot run any DDL from the workers unless you set citus.enable_ddl_propagation to off  ? So, that sounds expected to me

Reply:
Right, sorry about that. I confused the logic for a moment there. Let me change the PR